hijack/locale/de/LC_MESSAGES/django.po

```diff
@@ -37,4 +37,4 @@ msgstr "Ausblenden"
 
 #: templates/hijack/notification.html:18
 msgid "release"
-msgstr "freigeben"
+msgstr "Übernahme beenden"
```

Comment:
```suggestion
msgstr "beenden"
```

Reply:
Here I think the `beenden` is not super clear to "non-technical" users, so I would prefer the  `Übernahme beenden`. But as this notification template can anyway overriden locally,  it's also fine to change it to "beenden" & if needed overriding when you copy the notification.html template & use it.